src/eventable.js

```diff
@@ -82,11 +83,13 @@ function getEventableModule (notifyContext) {
       }
 
       const eventListeners = listeners[event]
-      if (eventListeners === undefined) return
+      if (!Array.isArray(eventListeners)) return
 
       // Traverse backwards and execute the newest listeners first.
       // Stop if a listener returns false.
-      eventListeners.reverse().every((listener) => listener.apply(context, args) !== false)
+      eventListeners
+      .slice().reverse() // reverse is also in-place, so copy array before
```

Comment:
The thing that's confusing me with reverse is that it's modifying the array in-place and also returning the reference.

Other in-place methods usually return something different: e.g. `[].push` returns the new index, `[].splice` returns the elements that it removed etc.

If `[].reverse` wouldn't return the reference this would've failed earlier. The way it's implemented it makes you believe it's not in-place.